client/src/components/Kanban/Board.tsx

```diff
@@ -1,17 +1,19 @@
 import { Grid } from '@material-ui/core';
 import { DragDropContext } from 'react-beautiful-dnd';
-import { useKanban } from '../../context/useKanbanContext';
+import { IBoard } from '../../interface/Board';
+import { useKanban } from '../../context/';
 import { FocusCard } from './FocusCard/FocusCard';
 import Column from './Column/Column';
 
-const Board = (): JSX.Element => {
-  const { columns, handleDragEnd } = useKanban();
+const Board = ({ activeBoard }: { activeBoard: IBoard }): JSX.Element => {
+  const { handleDragEnd } = useKanban();
```

Comment:
Remove columns from `KanbanContext` (it's only used now locally rather than an export)